api/app/src/routes/helpers/provider-route-helper.ts

```diff
@@ -43,6 +44,9 @@ export async function getProviderDataForType<T>(
       data.push(result.value);
     } else {
       console.error(result.reason);
+      capture.error(result.reason, {
```

Comment:
`result.reason` is `any` - it will "fit" anywhere. We need to be careful with it.

In this case, we're passing a "string" disguised as `any` to a function that accepts `any` (bc Errors are like that, unfortunately), which in turn gets sent to Sentry to a function that expects the error.

I don't know if Sentry deals with "string" well on the `.error()` function, but there's the `.message()` one if it doesn't.

Reply:
See here: https://metriport-inc.sentry.io/issues/4091929122/?environment=development&query=is%3Aunresolved&referrer=issue-stream&stream_index=0

Sentry seems to handle this fine, `reason` should be the underlying exception

Reply:
Yeah, the reason is the Promise that was rejected, so its the Error if it was rejected with an exception. Just confirmed. 👍